test/runner/lib/util.py

```diff
@@ -170,8 +170,8 @@ def raw_command(cmd, capture=False, env=None, data=None, cwd=None, explain=False
         encoding = 'utf-8'
         data_bytes = data.encode(encoding) if data else None
         stdout_bytes, stderr_bytes = process.communicate(data_bytes)
-        stdout_text = stdout_bytes.decode(encoding) if stdout_bytes else u''
-        stderr_text = stderr_bytes.decode(encoding) if stderr_bytes else u''
+        stdout_text = stdout_bytes.decode(encoding, 'replace') if stdout_bytes else u''
+        stderr_text = stderr_bytes.decode(encoding, 'replace') if stderr_bytes else u''
```

Comment:
@mattclay what do you think of this?  I think it's okay but I trace through and see how stdout_text and stderr_text are used later to make sure.